apps/client/src/features/app-settings/panel/project-panel/ProjectList.tsx

```diff
@@ -1,19 +1,42 @@
-import { IconButton, Menu, MenuButton, MenuItem, MenuList } from '@chakra-ui/react';
-import { IoEllipsisHorizontal } from '@react-icons/all-files/io5/IoEllipsisHorizontal';
+import { useMemo, useState } from 'react';
 
 import { useProjectList } from '../../../../common/hooks-query/useProjectList';
 import * as Panel from '../PanelUtils';
 
-import style from './ProjectPanel.module.scss';
+import ProjectListItem from './ProjectListItem';
+
+export type EditMode = 'rename' | 'duplicate' | null;
```

Comment:
todo: since this is used in the context of the list items, I think it should be exported from there

I am not sure what is the behaviour of imports that are not explicitly marked as `type`, but moving the export could also avoid circular imports here

Reply:
I moved the export to `ProjectListItem`